content/release-10-17-0/whatsnew-10-17-0.md

```diff
@@ -5,3 +5,44 @@ layout: bundle
 ---
 
 Release 10.17.0 includes the following new features or major feature enhancements.
+
+
+
+
+### Application enablement
+
+
+#### User interface redesign
+
+The entire user interface has undergone a complete overhaul, resulting in an improved, more modern and visually appealing design. Our goal was to enhance the user experience and make navigation easier and more intuitive. The redesign features a clean and sleek layout, with clearer typography and more intuitive navigation. Get ready to experience a whole new level of user-friendly design.
+
+![Application redesign](/images/release-notes/ux-redesign.png)
+
+
+#### Widgets
+
+##### Markdown widget
+We are proud to announce the release of Markdown, a new widget designed for displaying files written in markup language. With Markdown, you can now easily present your content in a clear and organized manner. Enhance your display capabilities by incorporating Markdown into your workflow.
+
+![Markdown widget](/images/release-notes/markdown-widget.png)
+
+##### Image widget
+
+We are pleased to announce that our Image widget has been improved with the addition of configuration options for image position and size. Now you have even more control over the display of your images.
+
+![Image widget](/images/release-notes/image-widget.png)
```

Comment:
Should we mention the migration to angular? but I guess it is not really of interest for someone reading the overall release notes..